lib/evss/disability_compensation_auth_headers.rb

```diff
@@ -30,15 +30,18 @@ def eauth_json
     end
 
     def gender
-      case @user.gender
```

Comment:
User.gender can only fall back to MVI when mhv_icn is present because we might need  to use gender to look up a person in MVI, but here we're free to fall back to MVI's gender.
``` ruby
  def gender
    identity.gender || (mhv_icn.present? ? mvi&.profile&.gender : nil)
  end
```
`